### PR TITLE
fix(yarn.lock): links to packages

### PR DESCRIPTION
**Pull request by promo360**
#8

### CODE REVIEW

File: packages/kl-runtime/yarn.lock [hunk 1]
```diff
@@ -1003,7 +1003,7 @@
 
 "@kl/dev-tools@2.9.0":
   version "2.9.0"
-  resolved "https://registry.npmjs.org/@kl/dev-tools/-/@kl/dev-tools-2.9.0.tgz#bd3b11d038326a0d17bbd270ce6ae5c9d30fde3c"
+  resolved "https://registry.npmjs.org/@kl/dev-tools/-/@kl/-/dev-tools-2.9.0.tgz#bd3b11d038326a0d17bbd270ce6ae5c9d30fde3c"
```
Comment (promo360) [inline packages/kl-runtime/yarn.lock:1006]: I'm not sure about @kl packages. But the old link is also not available.

Comment (vostrik) [reply packages/kl-runtime/yarn.lock:1006]: Yah, thank you! Today our second day after open source publication. I will check all builds

**Comment by promo360** [comment]
I see that the fix is planned in a different way (https://github.com/KasperskyLab/uif/pull/14). I am closing this pull request.

**Comment by vostrik** [comment]
@promo360 yah, I will turn into native package-lock and npm to reduce extra tooling around UIF. BTW thank a lot for your investigations — they helped a lot.

**Pull request closed without merging**